src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -435,4 +436,18 @@ describe('526 helpers', () => {
       expect(needsToEnter781({ formData })).to.be.false;
     });
   });
+
+  describe('transformRelatedDisabilities', () => {
+    it('should return an array of strings', () => {
+      const relatedDisabilities = {
+        'Some condition name': true,
+        'Another condition name': true,
+        'This condition is falsey!': false,
+      };
+      expect(transformRelatedDisabilities(relatedDisabilities)).to.eql([
+        'Some condition name',
+        'Another condition name',
+      ]);
+    });
+  });
```

Comment:
Do you think we should add some tests for the `transform` function in general?

Reply:
Unless...maybe the schema tests are sufficient?

Reply:
Yeah, I was gonna stick with the schema tests. I think testing the `transform` function itself might be redundant.